bothub/project/usecases/project/creation.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib.auth import get_user_model
+
+from .projectdto import ProjectCreationDTO
+
+from bothub.common.models import Organization
+from bothub.project.usecases.exceptions import InvalidProjectData
```

Comment:
Since `exceptions` is in the current package, you can do a relative import, like:
```py
from ..exceptions import InvalidProjectData
```